test/surface/read-logged_in.test.ts

```diff
@@ -245,5 +245,50 @@ describe('Read-LoggedIn', () => {
     const result = await solidLogicBob.fetch(resourceUrl);
     expect(result.status).toEqual(403);
   });
-
+  it('Is disallowed to read .acl with Read', async () => {
+    const resourceUrl = `${testFolderUrl}9/test.txt`;
+    // This will do mkdir-p:
+    const creationResult =  await solidLogicAlice.fetch(resourceUrl, {
+      method: 'PUT',
+      body: '<#hello> <#linked> <#world> .',
+      headers: {
+        'Content-Type': 'text/turtle',
+        'If-None-Match': '*'
+      }
+    });
+    const aclDocUrl = await solidLogicAlice.findAclDocUrl(resourceUrl);
+    await solidLogicAlice.fetch(aclDocUrl, {
+      method: 'PUT',
+      body: makeBody('acl:Read', 'acl:Read', resourceUrl),
+      headers: {
+        'Content-Type': 'text/turtle',
+        // 'If-None-Match': '*' - work around a bug in some servers that don't support If-None-Match on ACL doc URLs
+      }
+    });
+    const result = await solidLogicBob.fetch(aclDocUrl);
+    expect(responseCodeGroup(result.status)).toEqual("4xx");
+  });
+  it('Is allowed to read .acl with Control+Read', async () => {
```

Comment:
It should even be allowed with just Control. Read only applies to reading the resource itself.

Control access is everything (create, read, update, delete) or nothing for just the ACL doc associated with the resource.

Reply:
Right, changing this to Control only.